tests/tests.js

```diff
@@ -1,45 +1,193 @@
+
+    formatPriceSpy = sandbox.spy(pricing, 'formatPrice')
+    getEmployerContributionSpy = sandbox.spy(pricing, 'getEmployerContribution')
+    calculateVolLifePricePerRoleSpy = sandbox.spy(pricing, 'calculateVolLifePricePerRole')
+    calculateVolLifePriceSpy = sandbox.spy(pricing, 'calculateVolLifePrice')
+    calculateLTDPriceSpy = sandbox.spy(pricing, 'calculateLTDPrice')
+  })
+
+  afterEach(() => {
+    sandbox.restore()
+  })
+
+  describe('calculateLTDPrice', () => {
+    it('returns price for LTD plan for a single employee without employer contribution', () => {
+      const selectedOptions = {
+        familyMembersToCover: ['ee'],
+        coverageLevel: [
+          { role: 'ee', coverage: 125000 }
+        ],
+      }
+      const price = pricing.calculateLTDPrice(products.longTermDisability, employee, selectedOptions)
+
+      expect(price).to.equal(32.04)
+
+      expect(calculateLTDPriceSpy).to.have.callCount(1)
+    })
+
+  })
+
+  describe('calculateVolLifePrice', () => {
+    it('returns the price for a plan for single employee', () => {
+      const selectedOptions = {
+        familyMembersToCover: ['ee'],
+        coverageLevel: [
+          { role: 'ee', coverage: 125000 }
+        ],
+      }
+      const price = pricing.calculateVolLifePrice(products.voluntaryLife, selectedOptions)
+
+      expect(price).to.equal(43.75)
+
+      expect(calculateVolLifePricePerRoleSpy).to.have.callCount(1)
+    })
+
+    it('returns the price for a plan for an employee with a spouse', () => {
+      const selectedOptions = {
+        familyMembersToCover: ['ee', 'sp'],
+        coverageLevel: [
+          { role: 'ee', coverage: 200000 },
+          { role: 'sp', coverage: 75000 },
+        ],
+      }
+      const price = pricing.calculateVolLifePrice(products.voluntaryLife, selectedOptions)
+
+      expect(price).to.equal(79)
+
+      expect(calculateVolLifePricePerRoleSpy).to.have.callCount(2)
+    })
+  })
+
+  describe('calculateVolLifePricePerRole', () => {
+    it('returns the price for a single employee', () => {
+      const selectedOptions = {
+        familyMembersToCover: ['ee'],
+        coverageLevel: [
+          { role: 'ee', coverage: 125000 }
+        ],
+      }
+      const price = pricing.calculateVolLifePricePerRole(selectedOptions.coverageLevel[0].role, selectedOptions.coverageLevel, products.voluntaryLife.costs)
+
+      expect(price).to.equal(43.75)
+
+      expect(calculateVolLifePricePerRoleSpy).to.have.callCount(1)
+    })
+
+    it('returns the price for a spouse', () => {
+      const selectedOptions = {
+        familyMembersToCover: ['sp'],
+        coverageLevel: [
+          { role: 'sp', coverage: 85000 }
+        ],
+      }
+      const price = pricing.calculateVolLifePricePerRole(selectedOptions.coverageLevel[0].role, selectedOptions.coverageLevel, products.voluntaryLife.costs)
+
+      expect(price).to.equal(10.2)
+
+      expect(calculateVolLifePricePerRoleSpy).to.have.callCount(1)
```

Comment:
No need for this for the same reason as the ones above, the call is the one you made.